x-pack/plugins/elastic_assistant/server/routes/post_actions_connector_execute.ts

```diff
@@ -280,9 +289,12 @@ export const postActionsConnectorExecuteRoute = (
           logger.error(err);
           const error = transformError(err);
           telemetry.reportEvent(INVOKE_ASSISTANT_ERROR_EVENT.eventType, {
```

Comment:
I don't think this has anything to do with this PR, but when testing the error scenario (using a connector with bad creds), the error shows as a regular message instead of an error. Just noting since we have that other issue with persisting errors we need to fix:

<p align="center">
  <img width="700" src="https://github.com/elastic/kibana/assets/2946766/33bd1426-e8f6-4c4c-9b89-78f8cee000c8" />
</p>